integration_test/ops_agent_test.go

```diff
@@ -679,6 +679,50 @@ func TestModifyFields(t *testing.T) {
 	})
 }
 
+func TestResourceNameLabel(t *testing.T) {
+	t.Parallel()
+	gce.RunForEachPlatform(t, func(t *testing.T, platform string) {
+		t.Parallel()
+		ctx, logger, vm := agents.CommonSetup(t, platform)
+		file1 := fmt.Sprintf("%s_1", logPathForPlatform(vm.Platform))
+
+		config := fmt.Sprintf(`logging:
+  receivers:
+    f1:
+      type: files
+      include_paths:
+      - %s
+  processors:
+    json:
+      type: parse_json
+  exporters:
```

Comment:
[Optional] `exporters` are not required and can be inferred from the configuration if they are not specified.

Reply:
They're not only not required, but they're completely ignored if specified.